samcli/commands/build/command.py

```diff
@@ -113,7 +113,9 @@
     default=None,
     multiple=True,  # Can pass in multiple env vars
     required=False,
-    help="Path to environment variable json file (ex: env_vars.json) to pass into build containers",
+    help="Input environment variables through command line to pass into build containers, you can either"
```

Comment:
seems to miss a space at the end of the line.

Reply:
good catch!